modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm

```diff
@@ -60,11 +60,6 @@ GLOBAL_LIST_INIT(loadout_gloves, generate_loadout_items(/datum/loadout_item/glov
 	name = "Red Gloves"
 	item_path = /obj/item/clothing/gloves/color/red
 
-/datum/loadout_item/gloves/yellow
-	name = "Yellow Gloves"
-	item_path = /obj/item/clothing/gloves/color/yellow
```

Comment:
```suggestion
	name = "Budget Insulated Gloves"
	item_path = /obj/item/clothing/gloves/color/fyellow
```

Reply:
budgets can still be perfectly insulated

Reply:
fucking

Reply:
```
/obj/item/clothing/gloves/color/faker_yellow
	name = "yellow gloves"
	desc = "A pair of yellow gloves."
	icon_state = "yellow"
	inhand_icon_state = "ygloves"
```

Reply:
Went ahead and changed how i did it